config/routes.rb

```diff
@@ -67,7 +67,7 @@
   get "/beta-capabilities" => "publisher_information#beta_capabilities", as: :beta_capabilities
   get "/publisher-updates" => "publisher_information#publisher_updates", as: :publisher_updates
 
-  post "/govspeak-preview" => "govspeak_preview#to_html"
+  post "/documents/:document_id/govspeak-preview" => "govspeak_preview#to_html", as: :govspeak_preview
```

Comment:
This would be more consistent with our other document routes if it was "/documents/:id/govspeak-preview"

Reply:
Indeed - done.